File: Array of random numbers/C/arrayOfRand.c
#include <stdio.h>
#include <stdlib.h>
#define SIZE 100
int main(){
	int arrayLength, i, resultNumber;
	i = 0;
	char arrayOfRand[SIZE];
	printf("%s\n", "Кол-во случайных чисел:");
	scanf("%d", &arrayLength);
	while (i<arrayLength){
		resultNumber = i * rand()%100;;
		sprintf(arrayOfRand, "%s%s%d", arrayOfRand, " ", resultNumber);
		i = i + 1;
	}
	printf("%s\n", arrayOfRand);
}

File: Array of random numbers/Ruby/arrayOfRand.rb
arrayLength = gets()
arrayLength = arrayLength.to_i
i = 0;
arrayOfRand = Array.new(arrayLength)
while(i<arrayLength)
	resultNumber = i * rand(100)
	resultNumber = resultNumber.to_i
	arrayOfRand[i] = resultNumber
	i+=1
end
puts(arrayOfRand);